src/adapters/search/custom-search.js

```diff
@@ -0,0 +1,56 @@
+export default class MySearchAdapter {
```

Comment:
This file does not belong here. If this is a test, just place it within the test file.